features/other_b/skin_wizard.feature

```diff
@@ -12,6 +12,32 @@ Feature: Skin wizard
   Then I should see "Create New Skin"
     And I should see "CSS"
 
+  @javascript
+  Scenario: User can add a parent skin to the wizard
+  Given I am logged in
+    And I create the skin "Dad"
+  When I go to the new skin page
+    And I follow "Use Wizard"
+  Then I should see "Site Skin Wizard"
+    And I should see "Parent Skins"
+  When I fill in "Title" with "Child"
+    And I follow "Add parent skin"
+    And I wait 1 second
+  Then I should see a parent skin text field
+
+  @javascript
+  Scenario: User can add a parent skin to the form
+  Given I am logged in
+    And I create the skin "Dad"
+  When I go to the new skin page
+  Then I should see "Advanced"
+  When I follow "Show ↓"
+  Then I should see "Parent Skins"
+  When I fill in "Title" with "Child"
+    And I follow "Add parent skin"
+    And I wait 1 second
+  Then I should see a parent skin text field
```

Comment:
I wanted to see if we could test the whole process (i.e., filling in a parent skin name and saving), and here's what I came up with: 

```
  @javascript
  Scenario: User can add a parent skin to the form
  Given I am logged in
    And I create the skin "Dad"
  When I go to the new skin page
    And I fill in "Title" with "Child" # I moved this just to make it a little clearer that it's not related to adding the parent skin.
  Then I should see "Advanced"
  When I follow "Show ↓"
  Then I should see "Parent Skins"
  When I follow "Add parent skin"
    And it is currently 1 second from now
  Then I should see a parent skin text field
  When I enter "Dad" in the "skin_skin_parents_attributes_1_parent_skin_title_autocomplete" autocomplete field
    And I press "Submit"
  Then I should see "Parent Skins"
    And I should see Dad"
```

This approach should work for the wizard, too; I just used the regular skin form because I hadn't pulled the new code. 

What I did have to do to make this work is change the redirect to use `skin_path(@skin)` on line 80, like @tickinginstant  mentioned to you when you were working on series positions:

https://github.com/otwcode/otwarchive/blob/c7b3a865a605ae8419f2a88622574c179fe48b2b/app/controllers/skins_controller.rb#L62-L80